
--- We are now in /home/a.norouzikandelati/Projects/Tillage_mapping/codes/qsubs ...

--------- environment ---------
--------- where am i  ---------
WORKDIR:
HOMEDIR:
Running time on host cn1
Time is Sun Sep 29 08:38:15 PM PDT 2024
Directory is /home/a.norouzikandelati/Projects/Tillage_mapping/codes/qsubs
--------- continue on ---------
field_count = 32851
(31, 20)
field_count = 35071
(34, 16)
field_count = 66149
(65, 14)
Failed to delete doubleCropping_data (ID: 1aa0yaXU5TQXRb6H8RT5VTUfDxbR7bcYs): <HttpError 403 when requesting https://www.googleapis.com/drive/v3/files/1aa0yaXU5TQXRb6H8RT5VTUfDxbR7bcYs? returned "The user does not have sufficient permissions for this file.". Details: "[{'message': 'The user does not have sufficient permissions for this file.', 'domain': 'global', 'reason': 'insufficientFilePermissions'}]">
(31, 3428)
(34, 3428)

----- DONE -----

